simplified-ui-catalog/src/main/java/org/nypl/simplified/ui/catalog/CatalogFragmentBookDetail.kt

```diff
@@ -264,19 +264,23 @@ class CatalogFragmentBookDetail : Fragment() {
 
     this.bookRegistrySubscription =
       this.bookRegistry.bookEvents()
-        .subscribe(this::onBookStatusEvent)
+        .subscribe(this::onBookChanged)
```

Comment:
This method was renamed solely for consistency with the same method in `CatalogPagedViewHolder`.